Undefined name: 'ast_options' in scripts/make_ast_classes.py

Fixed in #2011

$ `flake8 --select=E9,F63,F7,F82 --show-source scripts/make_ast_classes.py`
```
scripts/make_ast_classes.py:33:47: F821 undefined name 'ast_options'
            ast_types[ast_type] = {'options': ast_options}
                                              ^
```

--- a/scripts/make_ast_classes.py
+++ b/scripts/make_ast_classes.py
@@ -19,6 +19,7 @@ with open('Python.asdl', 'wb') as out:
 
 f = open('Python.asdl', encoding='utf-8')
 type_def = False
+ast_options = []
 ast_types = {}
 ast_type = None
 

